pos_customer_required/__openerp__.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2004-Today Apertoso NV (<http://www.apertoso.be>)
+# Copyright (C) 2016-Today: La Louve (<http://www.lalouve.net/>)
+# @author: Jos DE GRAEVE (<Jos.DeGraeve@apertoso.be>)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html
+
+{
+    'name': 'Point of Sale Require Customer',
+    'version': '9.0.2.0.0',
```

Comment:
Version should be 9.0.1.0.0 (it's different from 8.0.1.0.0, so no problem with that).